docs/en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,99 @@
+   * Add a new issue in the "Backlog" on the [project board](https://github.com/silverstripe-security/security-issues/projects/1).
+   * Use the [CVSS Calculator](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator) to determine the issue severity
+   * Once the issue is confirmed, create a _draft security advisory_ against the relevant GitHub repository. This will allow you to request a CVE.
+   * Once a CVE has been assigned, respond to issue reporter and add it to the Github issue 
+   * Clarify who picks up and owns the issue (assign in Github).
+     The owner can be separate from the developer resolving the issue,
+     their primary responsibility is to ensure the issue keeps moving through the process correctly.
+
+### When developing a fix
+
+   * Ensure you're working on the oldest supported minor release branch of every supported major release (see [Supported Versions](#supported-versions))
+   * Move the issue into "In Progress" on the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+   * Add fixes on the [http://github.com/silverstripe-security](http://github.com/silverstripe-security) repo. Don't forget to update branches from the upstream repo.
+   * Ensure that all security commit messages are prefixed with the CVE. E.g. "[CVE-2019-001] Fixed invalid XSS"
+   * Get them peer reviewed by posting on security@silverstripe.org with a link to the Github issue
+
+### Before release (or release candidate)
+
+   * For issues rated "high" or "critical" (CVSS of >=7.0), post a pre-announcement to the [security pre-announcement list](https://groups.google.com/a/silverstripe.com/forum/#!forum/security-preannounce).
+     It should include a basic "preannouncement description" which doesn't give away too much,
+     the CVSS score as well as the CVE identifier.
+   * Create a draft page under [Open Source > Download > Security Releases](https://www.silverstripe.org/admin/pages/edit/show/794).
+     Populate it with the information from the [Github project board](https://github.com/silverstripe-security/security-issues/projects/1).
+   * Link to silverstripe.org security release page in the changelog.
+   * Move the issue to "Awaiting Release" in the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+
+### Perform release
+
+   * Public disclosure of security vulnerabilities need to happen in stable releases (not pre-releases)
+   * Merge back from [http://github.com/silverstripe-security](http://github.com/silverstripe-security) repos shortly at the release (minimise early disclosure through source code)
```

Comment:
Not sure what "merge back" and "shortly at the release" mean...?
Is this cherry-picking the commit shortly _before_ the release?